app/components/subcase/subcase-titles-panel/subcase-titles-panel-view.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="auk-u-mb-8">
+  <div class="auk-u-mb-2">
+    <div class="auk-o-flex auk-o-flex--vertical-center">
+      <h2 class="auk-overline" data-test-subcase-titles-type>
+        {{#if (not @subcase.showAsRemark)}}
+          {{t "note"}}
+        {{else}}
+          {{t "text"}}
+        {{/if}}
+      </h2>
+      {{#if @allowEditing}}
+        <div class="auk-u-ml-2">
+          <Auk::ButtonLink
+            data-test-subcase-titles-edit
+            {{on "click" @onClickEdit}}
+          >
+            {{t "edit"}}
+          </Auk::ButtonLink>
+        </div>
+      {{/if}}
+    </div>
+  </div>
+
+  <div class="auk-u-mb-2">
+    <h4 class="auk-h3">
+      <pre class="auk-u-text-pre-line">
+        {{@subcase.shortTitle}}
+      </pre>
+    </h4>
+  </div>
+
+  <div class="auk-u-mb-2">
+    <pre class="auk-o-flex--inline auk-u-text-pre-line">
+      {{@subcase.title}}
+    </pre>
+  </div>
+
+  {{#if (not @subcase.showAsRemark)}}
+    <div class="auk-u-mb-2">
+      {{#if @subcase.subcaseName}}
+        <Auk::Pill @skin={{this.getPillSkin}}>
```

Comment:
Het zou me verbazen als dit werkt. Je geeft een functie mee, waar `@skin` gewoon een object verwacht. Best omvormen naar (in de js-file) een task om `approved` te laden die in de constructor opgeroepen wordt in combinatie met een echte getter.